app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeListFragment.java

```diff
@@ -77,7 +77,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
     switch (item.getItemId()) {
       case R.id.menu_item_new_crime:
         Crime crime = Crime.builder()
-            .id(generateUniqueIdentifier())
+            .id(generateUniqueIdentifier().toString())
```

Comment:
[take a look it](https://github.com/Raizlabs/DBFlow/commit/71d878de09e9949f86edb98b61a9dc0096373289)